research/delf/INSTALL_INSTRUCTIONS.md

```diff
@@ -32,10 +32,7 @@ PATH_TO_PROTOC=`pwd`
 Install python library dependencies:
 
 ```bash
-sudo pip install matplotlib
-sudo pip install numpy
-sudo pip install scikit-image
-sudo pip install scipy
+pip install matplotlib numpy scikit-image scipy
```

Comment:
You are right, consistency should be the top priority. I think that I would prefer to remove sudo everywhere. Firstly sudo is setup dependant and most users who need it should be aware of it. Secondly sudo can screw some users over who quickly copy & paste the command and don't catch the sudo. The issue is that `sudo pip` always installs stuff into the main system, which is not always desired. Many ppl have a setup where `pip install` will install stuff into their home or a virtualenv.

The down-side of just pip is that for many users this will fail with permission error. Novice users might not immediately know what to do. So an additional command / hint might be required.

Tensorflows solution in the past was to cover many cases ( i.e. if you use virtualenv do ..., if you want to install to home do `pip install --user` and if you want to install to system do `sudo pip install`). I think this is the worst solution, since it makes it somewhat unreadable and most ppl know what they want when they see a valid pip command. In the [newest version](https://www.tensorflow.org/install#download-a-package) they put a simple pip command on prominent position and cover many other cases [somewhat deeper down in the docu](https://www.tensorflow.org/install/pip#2.-create-a-virtual-environment-recommended).

Anyway I am very happy to remove all other `sudos`. But it is your call to make a judgment what you prefer.